### PR TITLE
Lesson 3

### PR DESCRIPTION
**Pull request by igorslepenkov**

### CODE REVIEW

**Review by shuliakleonid — APPROVED** [review]

**Comment by shuliakleonid** [comment]
image
in this area should be text

**Comment by shuliakleonid** [comment]
[like this ](https://codepen.io/kod101/pen/gdjPaW)

**Comment by shuliakleonid** [comment]
and you may try to optimize your import in CSS files. Several times you use import fonts. Try creating one CSS file in HTML. Try and if you don't know how it does, then write me.

**Review by KseniaKotova — APPROVED** [review]

**Review by egkirpichev — APPROVED** [review]

**Review by AlexisKozlov — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,73 +1,112 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AXIT</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
```
Comment (shuliakleonid) [inline index.html:15]: try add one file css, try import in css

File: social.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    width: 29.38rem;
+    height: 31.25rem;
+}
+
+.main1__content-text {
+    margin-left: 6.25rem;
+}
+
+.main1__content-text h2 {
+    margin-top: 0;
```
Comment (shuliakleonid) [inline social.css:90]: refactor this

File: social.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    margin: 0;
+    padding: 0.63rem 0;
+    font-family: var(--main-font-family);
+    font-size: 14px;
+}
+
+.main1__buttons > .tab1 {
+    z-index: 2;
+    width: 12.56rem;
+    border-right: 1px solid rgb(198, 203, 205);
```
Comment (shuliakleonid) [inline social.css:60]: refactor this

**Review by shuliakleonid — APPROVED** [review]

File: social.css (hunk 2 above)
Comment (igorslepenkov) [reply social.css:90]: Ok, but what do you mean by "refactor"? How can i make this work the right way?

File: social.css (hunk 3 above)
Comment (shuliakleonid) [reply social.css:60]: image

File: social.css (hunk 2 above)
Comment (shuliakleonid) [reply social.css:90]: image

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**